shared/dfinity/dre.py

```diff
@@ -425,14 +425,20 @@ def get_blessed_replica_versions(self) -> list[str]:
             raise AirflowException("dre exited with status code %d", r.exit_code)
         return cast(list[str], json.loads(r.output)["value"]["blessed_version_ids"])
 
+    def get_elected_hostos_versions(self) -> list[str]:
+        r = self.run("get", "elected-hostos-versions", "--json", full_stdout=True)
```

Comment:
FYI: you could've used `dre registry` and then fetching `elected_host_os_versions`